documentation/docs/config-store/ConfigStore/ConfigStore.mdx

```diff
@@ -23,7 +23,7 @@ new ConfigStore(name);
 ### Parameters
 
 - `name` _: string_
```

Comment:
placeholder for now, similar wording around the `name` parameter should be applied to other service-linked resource types